.travis.yml

```diff
@@ -27,19 +37,19 @@ install:
   # Useful for debugging any issues with conda
   - conda info -a
   # Create conda test environment
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy pandas xarray netCDF4 pytest-cov pytest-ordering future
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION
   - conda activate test-environment
-  # Get latest coveralls from pip, not conda
-  - pip install coveralls
-  - pip install pytest-flake8
+  - while read requirement; do conda install --yes $requirement; done < requirements.txt
+  # Get dependencies not provided by conda from pip (testing dependencies)
+  - pip install -r test_requirements.txt
   # set up data directory
   - mkdir /home/travis/build/pysatData
   # install pysat
   - python setup.py install
 
-# command to run tests
-script:
- - pytest -vs --flake8 --cov=pysat/
-
 after_success:
  - coveralls --rcfile=setup.cfg
+
+allow_failures:
+  - name: 'docs'
+    python: '3.6'
```

Comment:
I think this needs to be part of the jobs 
matrix above.  See examples at https://docs.travis-ci.com/user/customizing-the-build/

Reply:
I didn't want to mess with your matrix, but yeah.  I'll switch to jobs and change this.